chart/compass/values.yaml

```diff
@@ -23,7 +23,7 @@ global:
       version: "PR-1460"
     director:
       dir:
-      version: "PR-1504"
+      version: "PR-1529"
```

Comment:
you've added logs to some of the resolvers involved in the token/cert resolution such as cert resolver and one time token resolver but there but there is also a lot of branching in tenant mapping handler (director rest api) as part of pairing flow that could benefit from info logging